egs/an4/s5/run.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+. cmd.sh
+. path.sh
+
+stage=1
+nj=1
+an4_root='/home/allen/data/an4'
```

Comment:
Since this data can easily be downloaded from the web, you should set up the script to download it automatically.  Look at Librispeech for an example - or "yesno", which is simpler and easier.  Does it have a stable URL somewhere?